libshaderc/src/shaderc_test.cc

```diff
@@ -535,26 +495,22 @@ TEST(CompileKinds, Geometry) {
          EndPrimitive();
        }
   )";
-  EXPECT_TRUE(CompilationSuccess(compiler.get_compiler_handle(), kGeoShader,
-                                 shaderc_glsl_geometry_shader));
+  EXPECT_TRUE(CompilationSuccess(kGeoShader, shaderc_glsl_geometry_shader));
 }
 
-TEST(CompileKinds, TessControl) {
-  Compiler compiler;
-  ASSERT_NE(nullptr, compiler.get_compiler_handle());
+TEST_F(CompileKindsTest, TessControl) {
+  ASSERT_NE(nullptr, compiler_.get_compiler_handle());
   const std::string kTCSShader =
```

Comment:
Should be kTCShader, for Tesselleation Control Shader?   What's the second S in kTCSShader?

Better to just write it out:  kTessControlShader